apps/docs/src/app/core/component-docs/checkbox/checkbox-docs.component.ts

```diff
@@ -1,42 +1,60 @@
-import { Component, OnInit, ViewChildren, ElementRef, QueryList } from '@angular/core';
+import {Component} from '@angular/core';
 
-import * as formHtml from '!raw-loader!./examples/checkbox-example.component.html';
-import * as formHtmlTsCode from '!raw-loader!./examples/checkbox-examples.component.ts';
-import * as formGroupInputHtml from '!raw-loader!./examples/checkbox-form-group-example.component.html';
-import * as formGroupInputTs from '!raw-loader!./examples/checkbox-form-group-example.component.ts';
-import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
-import { DocsSectionTitleComponent } from '../../../documentation/core-helpers/docs-section-title/docs-section-title.component';
-import { ActivatedRoute } from '@angular/router';
+import * as checkboxDefaultTsCode from '!raw-loader!./examples/checkbox-default-example.component.ts';
+import * as checkboxTristateTsCode from '!raw-loader!./examples/checkbox-tristate-example.component.ts';
+import * as checkboxCustomValuesTsCode from '!raw-loader!./examples/checkbox-custom-values-example.component.ts';
+import * as checkboxReactiveFormsTsCode from '!raw-loader!./examples/checkbox-reactive-forms-example.component.ts';
+import * as checkboxStatesTsCode from '!raw-loader!./examples/checkbox-states-example.component.ts';
+
+import {ExampleFile} from '../../../documentation/core-helpers/code-example/example-file';
 
 @Component({
     selector: 'app-input',
     templateUrl: './checkbox-docs.component.html'
 })
-export class CheckboxDocsComponent implements OnInit {
-    checkboxFormHtml: ExampleFile[] = [
+export class CheckboxDocsComponent {
+    checkboxDefault: ExampleFile[] = [
         {
-            language: 'html',
-            code: formHtml,
-            fileName: 'checkbox-example',
-            component: 'CheckboxExample',
-            secondFile: 'checkbox-examples',
-            typescriptFileCode: formHtmlTsCode
+            language: 'TypeScript',
+            fileName: 'checkbox-default-example',
+            component: 'CheckboxDefaultExamplesComponent',
+            code: checkboxDefaultTsCode
         }
     ];
 
-    checkboxFormGroup: ExampleFile[] = [
+    checkboxTristate: ExampleFile[] = [
         {
-            language: 'html',
-            code: formGroupInputHtml,
-            fileName: 'checkbox-form-group-example'
-        },
+            language: 'TypeScript',
+            fileName: 'checkbox-tristate-example',
+            component: 'DefaultCheckboxExamplesComponent',
+            code: checkboxTristateTsCode
+        }
+    ];
+
+    checkboxCustomValues: ExampleFile[] = [
         {
```

Comment:
You need to add the html as well for all

Reply:
Done